cypress/fixtures/stubbing/default/scenarios.js

```diff
@@ -35,25 +35,30 @@ export const DEFAULT_SCENARIOS_LIST = [
     ...SCENARIO_EXAMPLE,
     id: 's-stubbedscnr01',
     name: 'Test Cypress - Stubbed scenario 1',
+    depth: 0,
```

Comment:
This file contains samples of data **in the format sent by the Cosmo API**, you shouldn't add the `depth` key here :slightly_smiling_face:
If some tests are broken and you really need this `depth` key in the prop value provided to a component, you can either modify a copy of this list directly in the test, or create a new fixture in a new file, used specifically for the tests that need it.